### ISSUE TITLE
Allow a default editor for code blocks, including when a language isn't specified

### ISSUE THREAD

**Issue by FinlayBP** [mdx-editor/editor#114]
I'm currently running the code block and code mirror plugins as follows:

```js
codeBlockPlugin({ defaultCodeBlockLanguage: 'txt' }),
codeMirrorPlugin({
  codeBlockLanguages: {
    txt: 'text',
    js: 'JavaScript',
    ts: 'TypeScript',
    tsx: 'TypeScript (React)',
    jsx: 'JavaScript (React)',
    css: 'CSS',
    python: 'Python',
    bash: 'Bash',
  },
}),
```

But when users add empty backticks without a language specified:

```
` ` `
blah
` ` `
```

They currently recieve an error boundary because `Error: No CodeBlockEditor registered for language=null meta=null`. Perhaps  there could be a `defaultCodeBlockEditor` just like there's a `defaultCodeBlockLanguage`?

I imagine this would also remove the need to specify a large object of `codeBlockLanguages` if I want to use the `codeMirrorPlugin` for everything.

I had a go at the following as per the discussion in https://github.com/mdx-editor/editor/issues/95#issuecomment-1732225694, but `CodeMirrorEditor` isn't exported at the moment so this doesn't work:

```js
codeBlockEditorDescriptors: [
  {
    priority: 100,
    match: () => true,
    Editor: CodeMirrorEditor,
  },
],
```

**Comment by petyosi** [comment]
True, makes sense. Want to open a PR for the code mirror editor export? I feel like this is the easiest solution.
